docs/experimental-features.md

```diff
@@ -1,676 +0,0 @@
-# Experimental features of go-ipfs
```

Comment:
Some of these docs are actually informative, could we move all the less-frequently used into a `docs/old-ipfs` folder instead of deletion? Should only keep btfs-specific docs at top level

Reply:
Sure, I can do that. But for now, can I make a separate PR that re-uploads the files in a docs/old-ipfs folder? Thx